Examples/Modules/space_charge_initialization/analysis.py

```diff
@@ -12,14 +12,18 @@
 the expected theoretical field.
 """
 import sys
+
 import matplotlib
+
 matplotlib.use('Agg')
+import os
+
 import matplotlib.pyplot as plt
-import yt
 import numpy as np
-import os
 import scipy.constants as scc
+import yt
```

Comment:
Here again, with scipy being split

Reply:
Yes, because it sorts `import` and `from` separately.

We can configure this otherwise if we want to change that behavior:
https://github.com/PyCQA/isort/wiki/isort-Settings